book.toml

```diff
@@ -22,29 +22,29 @@ additional-css = [
 ]
 
 [preprocessor.readme2index]
-command = "./util/mdbook-readme2index"
+command = "./util/mdbook_readme2index.py"
```

Comment:
Can this commit be squashed into the individual commits where these preprocessors are introduced?

Reply:
Done